app/helpers/links_helper.rb

```diff
@@ -36,6 +36,8 @@ def link_agent_to_icon(link_agent)
     return :pawslut if link_agent.include? 'PawSlut'
     return :collin if link_agent.include? 'umbrella Collin\'s Walltaker Setter Thing'
     return :pawbies if link_agent.include? 'WTPopOutViewer'
+    return :ios_iphone if link_agent.include? 'WalltakeriOS-iPhone-Shortcut (mikapika.)'
+    return :ios_ipad if link_agent.include? 'WalltakeriOS-iPad-Shortcut (mikapika.)'
```

Comment:
You may also want to move your cases higher up in this list, above the :swift and :ioswidget cases, as I know iOS tends to append `widgetExtension` or `CFNetwork/` to your user agent depend on how the network call is made, which will cause your client to appear as one of those icons, not yours. (If that's the case! Don't do much native development myself in iOS)